doc/integrator/vector_tiles.rst

```diff
@@ -69,3 +69,9 @@ Here is an example SQL query template:
 Then your vector tiles will be accessible, for example in local development mode, through:
 
 https://localhost:8484/vector_tiles/{layer_name}/{z}/{x}/{y}.pbf
+
+.. note::
+
+   Currently vector tiles layers in GMF have a limited support. A vector tiles layer can be only
+   added as a background layer. It's not possible to print it and it doesn't support dimensions.
+   Queries can be only done via another wms layer using the metadata ``queryLayers``.
```

Comment:
Queries can be only done via another **WMS** layer using the metadata ``queryLayers``.

Reply:
Done see https://github.com/camptocamp/c2cgeoportal/pull/8856